test/test-fs-event.c

```diff
@@ -576,6 +576,7 @@ TEST_IMPL(fs_event_watch_file_exact_path) {
   create_dir("watch_dir");
   create_file("watch_dir/file.js");
   create_file("watch_dir/file.jsx");
+  uv_sleep(1000);
```

Comment:
Is this intentional? If the answer is 'yes' (which I think it is), can you add an explaining comment?

Reply:
Yes, some delay here seemed to be necessary on older versions of macos (pre-10.11.6): https://github.com/libuv/libuv/pull/2082#issuecomment-443371481 (to avoid getting notified of the creation of the files when creating the watcher below)

I'm not quite sure what the root cause is, or how best to deal with it for the test.

Reply:
I added a conditional guard here to the OS version and added a comment